UPDATING.md

```diff
@@ -41,6 +41,30 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Changes to airflow package imports
+
+Importing `executor`, `hooks`, `macros`, `operators`, `sensors` packages no
+longer works indirectly via airflow package. For example this will not work:
+
+```python
+import airflow
+
+operator = airflow.operators.BashOperator(...)
```

Comment:
Hasn't this alredy been deprecated/ I guess this did actuall owkr but issued a warning before this PR?

Reply:
I do not think there was a warning. I think it's not really possible (or easy and reliably) to have a warning depending on how you import class if it's location does not change. I don't think there is enough information on how the import happened when the __init__.py is executed for the package or when you reach the class.

Reply:
`airflow.operators.BashOperator` was relying on the "load all operators from submodule and stick it on operators module" behavoiur, and was already warned with the `Removed deprecated import mechanism` section in the UPDATING, which shows the same import paths you've got here.

So we might not need this here (though that could be updated to say it affects the other packages too)

Reply:
Ah yeah. Indeed. Removed it.